### PR TITLE
Add codacy code coverage; update to SCoverage 1.0.1

### PR DESCRIPTION
**Pull request by velvia**
Adds the codacy code coverage plugin and updates scoverage to 1.0.1.

NOTE: these are temporary steps to test out code coverage on codacy.  The long term answer is to add the codacy plugin to the autoplugins repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,12 @@ To build an assembly and run as a separate process:
 
 For soda-fountain-lib, log output goes to `sbt-test.log`.  Logging is controlled via `soda-fountain-lib/src/test/resources/log4j.properties`.
 
-For test coverage reports, do `sbt soda-fountain-lib/scoverage:test`.  XML and HTML coverage reports are generated.  Unfortunately due to a bug in scalac, HTML line by line highlighting is broken.
+For test coverage reports, do `sbt clean coverage test coverageReport`.  XML and HTML coverage reports are generated.  Unfortunately due to a bug in scalac, HTML line by line highlighting is broken.
```
Comment (kaidamasaki) [inline README.md]: Do you need `... clean coverage test coverageReport`?  I have xml and html reports being generated with just running `coverage test`.

**Comment by kaidamasaki** [comment]
LGTM.

File: README.md (hunk 1 above)
Comment (sokrahta) [inline README.md]: Kaida is correct: clean is included by dependency in coverage, and coverageReport is redundant

**Comment by velvia** [comment]
@kaidamasaki @sokrahta you guys are right!  Updated, may merge soon.

**Pull request merged**